backend/src/database_pg.ts

```diff
@@ -200,12 +248,99 @@ export class Database {
         }
     }
 
-    public async close() {
-        try {
```

Comment:
I think we still need a version of this to properly close connections. Did this logic move somewhere else?

Reply:
moved to logistics comment block